src/features/layer_features_panel/components/CardElements/CardImage.tsx

```
@@ -0,0 +1,13 @@
+import s from './CardImage.module.css';
+
+type CardImageProps = {
+  src: string;
+};
+
+export function CardImage({ src }: CardImageProps) {
+  return (
+    <div className={s.image}>
+      <img src={src} className={s.thumb}></img>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)